README.md

```diff
@@ -55,7 +55,7 @@ ActiveRecordDoctor::Rake::Task.new do |task|
   task.deps = []
 
   # A path to your active_record_doctor configuration file.
-  task.config_path = ::Rails.root.join(".active_record_doctor")
+  task.config_path = ::Rails.root.join(".active_record_doctor.rb")
```

Comment:
This section it titled "Additional Installation Steps for non-Rails Projects" but this is using a Rails specific function.

Reply:
Great catch! This needs to be fixed.

Reply:
I opened an issue to address that. See #165.